data/seriesRepository.go

```diff
@@ -762,17 +765,10 @@ func (r *SeriesRepository) GetTransformation(
 		if observationEnd.IsZero() || observationEnd.Before(observation.Date) {
 			observationEnd = observation.Date
 		}
-		dataPortalObservation := models.DataPortalObservation{
-			Date:  observation.Date,
-			Value: observation.Value.Float64,
-		}
-		if observation.PseudoHistory.Valid && observation.PseudoHistory.Bool {
-			dataPortalObservation.PseudoHistory = &observation.PseudoHistory.Bool
-		}
-		observations = append(
-			observations,
-			dataPortalObservation,
-		)
+		// This "magic" date must be used for formatting!
```

Comment:
We should probably resist the urge to comment about all the language idiosyncrasies, though I completely embrace the sentiment.